.gitignore

```diff
@@ -25,12 +25,23 @@ public/uploads/
 # Ignore application configuration
 /config/application.yml
 /config/puma.rb
+# Ignore deploy secrets
+/config/deploy.rb
+/config/deploy/*
+/Capfile
```

Comment:
This lines are no required because the app have not deploy secrets

Reply:
Done